scripts/infra/nvidia-setup.sh

```diff
@@ -47,12 +50,16 @@ FILEEOF
 
 rm -rf yum-packaging-precompiled-kmod
 dnf install libicu podman skopeo git rpm-build make openssl elfutils-libelf-devel python3.11 python3.11-devel -y
-if [ "${KERNEL_VERSION}" == "" ]; then \
-      RELEASE=$(dnf info --installed kernel-core | sort | awk -F: '/^Release/{print $2}' | tr -d '[:blank:]' | tail -n 1) \
-      && VERSION=$(dnf info --installed kernel-core | sort | awk -F: '/^Version/{print $2}' | tr -d '[:blank:]' | tail -n 1) \
-      && export KERNEL_VERSION="${VERSION}-${RELEASE}" ;\
-fi \
-    && dnf install -y "kernel-devel-${KERNEL_VERSION}" \
+if [ "${KERNEL_VERSION}" == "" ]; then
```

Comment:
nit: I realize you didn't change it but would this be better as -z "${KERNEL_VERSION}"